JDTeamAcetabulum/qna
# Creating short answer question Fixed

## Changes 
Users are able to create Short Answer Question now

## Context 

## Testing 
- Create a Short answer question
- Publish it
- Check from student view

## Pre-merge checklist 

- [ ] Build succeeds locally and in CI
- [ ] Proper documentation
- [ ] Git history clean
